### PR TITLE
[ch54] Connect FE dev app to API

### PR DESCRIPTION
**Pull request by seerocode**
Story details: https://app.clubhouse.io/stanford-computational-policy-lab/story/54

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -124,5 +124,6 @@
     "ts-jest": "^26.5.3",
     "tslint": "^6.1.3",
     "tslint-plugin-prettier": "^2.3.0"
-  }
-}
+  },
+  "proxy": "http://localhost:8000"
```
Comment (nmanduj1) [inline client/package.json:128]: Hm, I'd expect this info to be in the apollo configuration/initializer, not in package.json. But I'm not sure how the proxy is being used here. Are you able to hit this endpoint successfully?

Comment (nmanduj1) [reply client/package.json:128]: I just read the docs and it looks like you are completely on point- this the correct implementation for this proxy. I usually just add a flag to route traffic depending on whether the node_env is set to development. Do you know what the pros are to setting this up this way?

Comment (seerocode) [reply client/package.json:128]: @nmanduj1 yep! This is being brought in with the Apollo Client in a different PR.

Comment (seerocode) [reply client/package.json:128]: @nmanduj1 From the docs I understood this is for development and avoids having to change your host everywhere when you deploy to production. There is [middleware](https://create-react-app.dev/docs/proxying-api-requests-in-development/#configuring-the-proxy-manually) that provides additional flexibility should we need it or we can set up env files. This seemed to be a good start although I considered the middleware.

Comment (nmanduj1) [reply client/package.json:128]: thank u!

**Review by nmanduj1 — APPROVED** [review]

**Pull request merged**